src/projection/node/create-projection-node.ts

```diff
@@ -604,8 +604,8 @@ export function createProjectionNode<I>({
         updateSnapshot() {
             if (this.snapshot || !this.instance) return
             const measured = this.measure()!
-            const layout = this.removeElementScroll(
-                this.removeTransform(measured)
+            const layout = this.removeTransform(
```

Comment:
We could maybe put a comment here in case the order is changed by mistake later?